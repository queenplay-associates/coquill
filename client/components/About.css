.about {
    display: flex;
    margin-left: 20%;
    align-self: right;
    flex-wrap: wrap;
    max-width: 30%;
    justify-content: flex-end;
}

/*img.about-img {*/
    /*display: flex;*/
    /*align-self: left;*/
    /*max-width: 50%;*/
/*}*/
img.carousel-image {
    max-height: 950px;
}
