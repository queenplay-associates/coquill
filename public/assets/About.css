@import url('https://fonts.googleapis.com/css?family=Questrial');

.about-container {
    margin-top: 20%;
    margin-left: 50%;
    float: right;
}

.about {
    max-width: 70%;
    text-align: justify;
}

.start-button {
    font-family: Questrial, sans-serif;
    font-size: 25px;
    color: #615858;
    padding: 0.7rem 0.9rem;
    margin-left: 17%;
}

.logo-container {
    position: relative;
    margin-left: 10%;
}

.coquill {
    font-family: Trattatello, Helvetica, sans-serif;
    justify-content: center;
    font-size: 80px;
    position: absolute;
    color: #615858;
    padding-top: 15%;
    margin-top: 12%;
}

img.about-img {
    display: flex;
    align-self: left;
    max-width: 28%;
    position: absolute;
    padding-left: 10.5%;
    margin-top: 9%;
}
