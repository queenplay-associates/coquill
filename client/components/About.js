import React from 'react'
import { Link } from 'react-router-dom';

const carouselImages = [
  'https://s-media-cache-ak0.pinimg.com/originals/da/70/b9/da70b95ca65d402ce40ff8cdcd25d335.jpg',
  'https://quilljs.com/assets/images/features/cross-platform.svg',
  'https://quilljs.com/assets/images/features/scale.svg'
];

const logo = 'https://img00.deviantart.net/3b69/i/2015/125/6/f/burnt_quill_s_cutie_mark_final_version_by_burntquill-d8j1is9.png';

import '~/public/assets/About.css'

export default () =>
  <div>
    <div className="logo-container">
      <div className="coquill">COQUILL</div>
      <img className="about-img" src={logo} alt="logo" /> 
    </div>
    <div className='about-container'>
      <p className='about'>
        Coquill is a real-time collaborative text editor for writing screenplays. 
        Based on your selection, the editor will automatically format your text 
        and provide text styling if need be. Once you have created a screenplay, the editor 
        will project colors on the page which map to the context of your text.
      </p>
    <Link className='startbutton' to='/screenplays'>Get Started Now!</Link>
    </div>
  </div>
