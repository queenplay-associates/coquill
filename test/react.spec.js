
// function sum(a, b) {
//     return a + b;
//   }

// test('adds 1 + 2 to equal 3', () => {
//     expect(sum(1, 2)).toBe(3);
//   });


import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme,{ shallow, mpunt } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16';
import { Link } from 'react-router-dom';
import { MemoryRouter } from 'react-router'
import About from '../client/components/About'

Enzyme.configure({ adapter: new Adapter()});
//FIXME: takout __mock__ stuff. ðŸ™„

describe('<About /> component', () => {
  
  const about = renderer.create(<MemoryRouter>
    <About />  
  </MemoryRouter>)

  test('snapshop of the DOM', () => { 
    let tree = about.toJSON()
    expect(tree).toMatchSnapshot()
  })

  test('links <Link to />', () => {
    let wrapper = shallow(<About />)
    //let linka = wrapper.find(Link) //.find({to: '/screenplays'})
    //console.log(linka)
    expect(wrapper.find(Link).length).toBe(1);
  })
})
