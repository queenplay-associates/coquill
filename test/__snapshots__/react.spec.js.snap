// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> component snapshop of the DOM 1`] = `
<div>
  <div
    className="logo-container"
  >
    <div
      className="coquill"
    >
      COQUILL
    </div>
    <img
      alt="logo"
      className="about-img"
      src="https://img00.deviantart.net/3b69/i/2015/125/6/f/burnt_quill_s_cutie_mark_final_version_by_burntquill-d8j1is9.png"
    />
  </div>
  <div
    className="about-container"
  >
    <p
      className="about"
    >
      Coquill is a real-time collaborative text editor for writing screenplays. Based on your selection, the editor will automatically format your text and provide text styling if need be. Once you have created a screenplay, the editor will project colors on the page which map to the context of your text.
    </p>
    <a
      href="/screenplays"
      onClick={[Function]}
    >
      <button
        className="start-button"
      >
        Get Started Now!
      </button>
    </a>
  </div>
</div>
`;
